### ISSUE TITLE
0-RTT and Early Data

### ISSUE THREAD

**Issue by MikeBishop** [quicwg/ops-drafts#227]
Section 3 talks about the challenges of 0-RTT, but seems to ignore that very similar challenges exist when TLS 1.3 Early Data is used over TCP.  I think I'd reword this to talk about the similarity, and that while QUIC gives Early Data more visibility, the same cautions and mitigations still apply.

(I'd also probably find a more technical title than "Here There Be Dragons," but that's purely stylistic.)

**Comment by mirjak** [comment]
I agree that challenges are similar to TLS1.3 over TCP. However, given this document is scoped to the use of QUIC, I'm not sure what mentioning TCP in this section gets you. I find the section actually quite clear without mentioning anything about TCP. Or did I miss the point you are making?

**Comment by mirjak** [comment]
Otherwise PRs are always welcome! :-)
